tests/gatling_kickoff.rb

```diff
@@ -1,6 +1,26 @@
-step "Execute gatling scenario" do
-  on metric, "cd /root/gatling-puppet-load-test/simulation-runner/ && " +
-      "PUPPET_GATLING_MASTER_BASE_URL=https://#{master.hostname}:8140 " +
-      "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/ApplesToApples.json " +
-      "PUPPET_GATLING_SIMULATION_ID=PerfTestLarge sbt run"
+require 'beaker-benchmark'
+
+test_name 'Kickoff Gatling' do
+
+  atop_log = '/tmp/atop.log'
```

Comment:
This name being generic could cause collisions betweens runs launched from teh same testrunner... as beaker-benchmark copies to the test runner.